torchvision/csrc/cuda/ROIPool_cuda.cu

```diff
@@ -75,34 +73,35 @@ __global__ void RoIPoolFForward(const int nthreads, const T* bottom_data,
 }
 
 template <typename T>
-__global__ void RoIPoolFBackward(const int nthreads, const T* top_diff,
+__global__ void RoIPoolBackward(const int nthreads, const T* top_grad,
     const int* argmax_data, const int num_rois, const T spatial_scale,
     const int channels, const int height, const int width,
-    const int pooled_height, const int pooled_width, T* bottom_diff,
-    const T* bottom_rois) {
-  CUDA_1D_KERNEL_LOOP(index, nthreads) {
-    // (n, c, ph, pw) is an element in the pooled output
-    int pw = index % pooled_width;
-    int ph = (index / pooled_width) % pooled_height;
-    int c = (index / pooled_width / pooled_height) % channels;
-    int n = index / pooled_width / pooled_height / channels;
-
-    const T* offset_bottom_rois = bottom_rois + n * 5;
-    int roi_batch_ind = offset_bottom_rois[0];
-    int bottom_offset = (roi_batch_ind * channels + c) * height * width;
-    int top_offset    = (n * channels + c) * pooled_height * pooled_width;
-    const T* offset_top_diff = top_diff + top_offset;
-    T* offset_bottom_diff = bottom_diff + bottom_offset;
-    const int* offset_argmax_data = argmax_data + top_offset;
-
-    int argmax = offset_argmax_data[ph * pooled_width + pw];
-    if (argmax != -1) {
-      atomicAdd(
-          offset_bottom_diff + argmax,
-          static_cast<T>(offset_top_diff[ph * pooled_width + pw]));
-
+    const int pooled_height, const int pooled_width, T* bottom_data,
```

Comment:
I find this `bottom_data` name misleading, because it is actually the gradient of the `bottom`, so I think that `bottom_diff` was a better name, and more closely aligns with what was implemented in C2

Reply:
I actually wanted to update the names to be more torch-like e.g. `input`, `output`, `grad_input` and `grad_output`, since that would lead to minimum confusion. Should I just do that?